### PR TITLE
Herzlich willkommen bei holiday check

### PR DESCRIPTION
**Pull request by konarkmodi**

### CODE REVIEW

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (tmayer) [inline _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md:38]: tools that are built on Open Source

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (tmayer) [inline _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md:58]: will be transcribed as follows

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (tmayer) [inline _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md:79]: remove the majority of the personal information without losing too many bits of the conversion

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (tmayer) [inline _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md:82]: did you also commit the presidio image? I can't find it in the list of files

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (tmayer) [inline _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md:110]: I would also add a section about evaluating the outcome and how we keep track that the summary/translation is good enough

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (tmayer) [inline _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md:126]: do we want to mention that we explore such things in-house but then also investigate if there's a solution for that already existing and outsource it?

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (konarkmodi) [reply _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md:111]: Addes as "step 4"

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (konarkmodi) [reply _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md:83]: Eagle eyes. Added now.

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (konarkmodi) [reply _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md]: Done

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (konarkmodi) [reply _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md]: Done

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (konarkmodi) [reply _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md:35]: Done, added in the title and intro paragraph too.

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (konarkmodi) [reply _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md]: Done. replaced it with advancements in the field of AI

File: _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md
Comment (konarkmodi) [reply _posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md]: done

**Review by tmayer — APPROVED** [review]
LGTM Nice blog post!

**Comment by konarkmodi** [comment]
thanks @tmayer . Waiting for some internal reviews, will merge once I get a go ahead.

**Pull request merged**
